src/include/meta.h

```diff
@@ -98,93 +97,3 @@ struct __wt_blkincr {
                               /* AUTOMATIC FLAG VALUE GENERATION STOP 8 */
     uint8_t flags;
 };
-
-/*
- * Block modifications from an incremental identifier going forward.
- */
-/*
- * At the default granularity, this is enough for blocks in a 2G file.
- */
-#define WT_BLOCK_MODS_LIST_MIN 128 /* Initial bits for bitmap. */
```

Comment:
This should not be moved imo

Reply:
It should be part of the src/block module as it's used there.

Reply:
Perhaps `WT_BLOCK_MODS` is not ideally named. But the `WT_BLOCK_MODS` structure is only used inside the `WT_CKPT` structure. That is why they were defined in the same place. It is the structure for the checkpoint block modification bitmap for backup.

Reply:
should this have ckpt/checkpoint in the name?

Reply:
@etienneptl  That is calculating the bitmap that is stored in the `WT_BLOCK_MODS` structure. The define is only used in relation to that structure so they were defined together. We record the bitmap changes when we process blocks in `block_ckpt.c`. The structure could be renamed to `WT_CKPT_BLOCK_MODS` if you really wanted to do so.

Reply:
I moved it to the only place where it's used.